### PR TITLE
code cleaning

### PR DESCRIPTION
**Pull request by stadnytskyivp**

### CODE REVIEW

File: src/test/java/client/PetClient.java [hunk 1]
```diff
@@ -15,12 +14,17 @@
...
 
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.*;
 
 public class PetClient extends AbstractTest {
 
+    final private static String PET_PARAMETER = "/v2/pet/";
+    final private static String PET_FIND_BY_STATUS_PARAMETER = "/v2/pet/findByStatus?status=";
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: You can update url with `PET_PARAMETER + "findByStatus?status="`

File: src/test/java/client/PetClient.java [hunk 2]
```diff
@@ -15,12 +14,17 @@
...
 import pet.AbstractTest;
 
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.*;
 
 public class PetClient extends AbstractTest {
 
+    final private static String PET_PARAMETER = "/v2/pet/";
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: Consider renaming these parameters to something like `PET_ENDPOINT`, `FIND_PET_BY_STATUS_ENDPOINT` etc

File: src/test/java/client/PetClient.java [hunk 3]
```diff
@@ -40,6 +44,15 @@ public static RequestSpecification buildReq() throws IOException {
             .build();
     }
 
+    @Step("Building request specification")
+    public static RequestSpecification buildUncheckedReq() throws IOException {
+        LOGGER.debug("building request specification ");
+
+        return new RequestSpecBuilder()
+            .setBaseUri(getBaseUrl("PET_STORE_HOST"))
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: Do you ever use any other hosts? Consider removing input parameter for `getBaseUrl()` method altogether

File: src/test/java/client/PetClient.java [hunk 4]
```diff
@@ -187,4 +198,31 @@ public static Response deleteNonExistingPetById(long petId) throws IOException {
             .as(Pet[].class));
     }
 
+    @Step("Adding pet photo to the pet")
+    public static ResponseInfo postPetPicture(Long petId) throws IOException {
+
+        Formatter formatter = new Formatter();
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: This variable is only used once, you can call `new Formatter()` directly

File: src/test/java/client/PetClient.java [hunk 5]
```diff
@@ -187,4 +198,31 @@ public static Response deleteNonExistingPetById(long petId) throws IOException {
...
+
+        LOGGER.debug("sending request");
+
+        RequestSpecification res = RestAssured.given()
+            .spec(buildUncheckedReq())
+            .multiPart("file",new File(System.getProperty("user.dir") +
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: Missed spacing after comma

File: src/test/java/client/PetClient.java [hunk 6]
```diff
@@ -187,4 +198,31 @@ public static Response deleteNonExistingPetById(long petId) throws IOException {
...
+            .then()
+            .statusCode(HttpStatus.SC_OK)
+            .spec(buildUncheckedRes())
+            .log()
+            .body()
+            .extract()
+            .response()
+            .as(ResponseInfo.class);
+    }
+
```
Comment (andyyarych) [inline src/test/java/client/PetClient.java]: Redundant empty line

File: src/test/java/client/PetClient.java (hunk 3 above)
Comment (stadnytskyivp) [reply src/test/java/client/PetClient.java:52]: should I put this method it in petClient class?

**Review by andyyarych — APPROVED** [review]

**Pull request merged**